Assets/Scripts/ROS Scripts/ROS Connections v2/ROSManager.cs

```diff
@@ -12,15 +12,18 @@ public class ROSManager : MonoBehaviour {
 
     // Let's gooooo
 
+    // Drone Types & Subscribers supported by system
```

Comment:
Can we try using C# documentation comments? It will show up on our IDEs in the future which is nice.

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments

Reply:
Also I think maybe its clear enough in this case so not strictly necessary.